x-pack/platform/plugins/shared/onechat/public/services/agents/profiles/agents_profiles_service.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { HttpSetup } from '@kbn/core-http-browser';
+import type { AgentProfile } from '@kbn/onechat-common';
+import type {
+  AgentProfileCreateRequest,
+  AgentProfileUpdateRequest,
+  AgentProfileListOptions,
+} from '../../../../common/agent_profiles';
+
+export class AgentProfilesService {
+  private readonly http: HttpSetup;
+
+  constructor({ http }: { http: HttpSetup }) {
+    this.http = http;
+  }
```

Comment:
Now that we know those are going to be "agents", I'm torn between using that dedicated service or just having the APIs defined in the existing browser-side `AgentService`. For consistency, given we do have an agent profile service on the server atm, we should probably stick with it, and refactor both at the same time later.

So TLDR: let's keep it that way for now.